docs/_docs/monitoring-metrics/metrics.adoc

```diff
@@ -308,6 +308,22 @@ group=<Cache Name>,name="org.apache.ignite.internal.processors.cache.CacheLocalM
 |===
 --
 
+=== Monitoring Processed Keys for Rebuilding Indexes
+
+To get an estimate on how long it takes to rebuild cache indexes, you can use one of the metrics listed below:
+
+. `IsIndexRebuildInProgress` - gives the number of currently rebuilding cache indexes
+. `IndexBuildCountPartitionsLeft` - gives the remaining number of partitions (by cache group) to rebuild indexes for
+
+These metrics allow to estimate an approximate number of indexes left to rebuild.
+For a more accurate estimate, use the `IndexRebuildKeyProcessed` cache metric:
+
+* Use `org.apache.ignite.mxbean.CacheMetricsMXBean#isIndexRebuildInProgress` to know whether the indexes are being rebuilt for the cache.
+Note that presently only the `org.apache.ignite.internal.processors.cache.CacheLocalMetricsMXBeanImpl#isIndexRebuildInProgress` local metric is available.
+
+* Use `org.apache.ignite.mxbean.CacheMetricsMXBean#getIndexRebuildKeysProcessed` to know the number of indexes that are rebuilt for the cache. If the rebuilding is in progress, it gives a number of indexes being rebuilt at the current moment. Otherwise, it gives a total number of the rebuilt indexes. The values are reset before the start of each rebuilding.
```

Comment:
Not the number of indexes, but the number of keys for which all their indexes have been rebuilt.

Reply:
Fixed.